analytics_data_api/tests/test_middleware.py

```diff
@@ -0,0 +1,19 @@
+from django.conf import settings
+
+from analytics_data_api.middleware import thread_data
+from analytics_data_api.v0.tests.views import CourseSamples
+from analyticsdataserver.tests.utils import TestCaseWithAuthentication
+
+
+class RequestVersionMiddleware(TestCaseWithAuthentication):
+    def test_request_version_middleware_v1(self):
+        self.authenticated_get('/api/v1/courses/{}/activity'.format(
+            CourseSamples.course_ids[0]))
+
+        self.assertEqual(thread_data.analyticsapi_database, getattr(settings, 'ANALYTICS_DATABASE_V1'))
```

Comment:
Please add a test case to test the case where the setting `ANALYTICS_DATABASE_V1` is missing values